### ISSUE TITLE
Using own .cpp functions

### ISSUE THREAD

**Issue by nastya97** [kyamagu/mexopencv#298]
Hi,

I have created own mexFunction myFunc.cpp and placed it into src/+cv/myProject folder. I have compiled this folder successfully and can use this functions in .m files. However I don't know how to use myFunc.cpp in another myFunc2.cpp function. Could somebody explain me how to do that. Thank you.

**Comment by amroamroamro** [comment]
Any MEX-function you compile is called just like any other MATLAB function. So calling `cv.myFunc` is no different than calling say [`randn`](https://www.mathworks.com/help/matlab/ref/randn.html).

If you're working in C/C++, you'll have to use [`mexCallMATLAB`](https://www.mathworks.com/help/matlab/apiref/mexcallmatlab.html) to call other MEX-functions (or any function on the MATLAB side).

Obviously I'm talking about MEX-functions here, i.e shared libraries that expose a single `mexFunction` with the signature:
``` cpp
void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[]);
```
In other words, the inputs and outputs are `mxArray` pointers.
